### ISSUE TITLE
Import/export

### ISSUE THREAD

**Issue by vimeitor** [lakesare/memcode#2]
Thank you for making this project. I haven't used it much, but the little I've used, it's worked very well.

I think it would be a nice feature to be able to import/export flashcards/courses. That would make it easier to upload courses from memrise courses or anki decks.

**Comment by vimeitor** [comment]
Thank you, @lakesare!

I have tried it and it works pretty well. Is it possible that it doesn't pick up the first row of the document? I'm exporting from libreoffice to xls, so it could be a bug on my side.

**Comment by lakesare** [comment]
@vimeitor, yes, it keep the first row on purpose, because I'd expect the first row to be titles.

**Comment by vimeitor** [comment]
That was my theory. Will keep that in mind. Thanks.
